cmd/kops/completion.go

```diff
@@ -60,7 +60,7 @@ func NewCmdCompletion(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "completion",
-		Short:   "Output shell completion code for the given shell (bash)",
+		Short:   "Output shell completion code for the given shell (bash).",
 		Long:    longDescription,
```

Comment:
This is what is it does, but does not help me understand as a n00b, how it helps me